app/templates/power-ups/topics.html

```diff
@@ -192,50 +218,37 @@ <h3 id="internationalization">Internationalization</h3>
-    object from any of the interfaces or callbacks as well.</p>
+  <p>Once a localizer has been initialized, you can call the following new handlers (which behind the scenes make use of the localizer). We make the current user locale available to you via window.locale. Additionally, you can grab it off the options object from any of the interfaces or callbacks as well.</p>
   <p><strong>localizeKey(key, data)</strong> synchronously returns the output of window.localizer.localize(key, data)</p>
   <p><strong>localizeKeys(key)</strong> synchronously localizes multiple keys. Input can be either an array of keys ['key1',
     'key2'] or an arrary of key, data arrays [['key1', data1], ['key2', data2]]</p>
-  <p><strong>localizeNode(node)</strong> synchronously inserts localized texts into DOM nodes starting at the provided node and
-    all of it's children. Tag a node in your HTML with data-i18n-id="key" for its text content to be replaced based on that
-    key. You can pass args with data-i18n-args='{ "arg1": "data" }'. You can also have placeholder text replaced with data-i18n-attrs='{
-    "placeholder": "key" }'. This will also use the args defined in data-i18n-args.</p>
+  <p><strong>localizeNode(node)</strong> synchronously inserts localized texts into DOM nodes starting at the provided node and all of it's children. Tag a node in your HTML with data-i18n-id="key" for its text content to be replaced based on that key. You can pass args with data-i18n-args='{ "arg1": "data" }'. You can also have placeholder text replaced with data-i18n-attrs='{ "placeholder": "key" }'. This will also use the args defined in data-i18n-args.</p>
   <p>Currently there is no support for dot notation when replacing args.</p>
   <p>Argument replacement assumes that your resource file follows this format:</p>
   <pre>{
   "key": "value with replaceable {arguments}"
 }</pre>
-  <p>Again, dot notation is not supported for arguments. This also limits how you can use braces {} in that there is no escaping
-    currently supported.</p>
+  <p>Again, dot notation is not supported for arguments. This also limits how you can use braces {} in that there is no escapingcurrently supported.</p>
+  <hr/>
 
+  <h2 id="callbacks">Callbacks</h2>
   <h3>What are callbacks?</h3>
-  <p>Sometimes a Power-Up wants the web client to create a bit of UI, and the Power-Up wants to know when the user interacts
-    with that UI (e.g. when the user clicks on a button) For example, the Trello web client might run card-button on a Power-Up,
-    and the Power-Up could respond with</p>
+  <p>Sometimes a Power-Up wants the web client to create a bit of UI, and the Power-Up wants to know when the user interacts with that UI (e.g. when the user clicks on a button) For example, the Trello web client might run card-button on a Power-Up, and the Power-Up could respond with</p>
```

Comment:
Thanks for mentioning that. I cleaned it up a little bit. Let me know if that is still unclear.